scipy/optimize/_optimize.py

```diff
@@ -2248,12 +2248,19 @@ def get_bracket_info(self):
             if (xa > xc):  # swap so xa < xc can be assumed
                 xc, xa = xa, xc
             if not ((xa < xb) and (xb < xc)):
-                raise ValueError("Not a bracketing interval.")
+                 raise ValueError(
+                    "Bracketing values (xa, xb, xc) do not"
+                    "fulfill this requirement: (xa < xb) and (xb < xc)"
+                )
             fa = func(*((xa,) + args))
             fb = func(*((xb,) + args))
             fc = func(*((xc,) + args))
             if not ((fb < fa) and (fb < fc)):
-                raise ValueError("Not a bracketing interval.")
+                 raise ValueError(
```

Comment:
You have a few indentation issues. `raise` should be unindented by 2 spaces. Here and elsewhere.

Reply:
Okay, thank you so much. I actually did not know that I resolved the conversation. I was just committing your changes. Does committing your changes, resolve the conversation automatically?

Reply:
Ah ok, all good then. Yes some discussions are automatically resolved.